man/fi_domain.3.md

```diff
@@ -772,6 +773,12 @@ This specifies the default traffic class that will be associated any endpoints
 created within the domain.  See [`fi_endpoint`(3)](fi_endpoint.3.html)
 for additional information.
 
+## Max Authorization Keys per Endpoint (max_ep_auth_key_cnt)
+
+: The maximum number of authorization keys which can be supported per endpoint. If
+connectionless endpoints are supported and this value is non-zero, providers must
+support FI_AV_AUTH_KEY.
```

Comment:
I think we can state that this value only applies to connectionless endpoints.

nit: indentation needed for lines after ':'